code/__HELPERS/roundend.dm

```diff
@@ -228,6 +228,9 @@
 	if(mode.allow_persistence_save)
 		SSpersistence.CollectData()
 
+	///Collect Antag data
+	antagrounds()
```

Comment:
Is it possible to do this when someone is given antag status instead of trying to figure out who had antag status at round end?